example_data/sample_bundle.json

```diff
@@ -20,86 +30,90 @@
             "text": "NHS UK Healthcare Provider"
           }
         ],
-        "name": "Great Ormond Street Hospital for Children NHS Foundation Trust",
+        "name": "North London Lab 1",
+        "telecom": [
+          {
+            "system": "phone",
+            "value": "2078298870"
+          }
+        ],
         "address": [
           {
-            "line": [
-              "Level 4 Barclays House",
-              "37 Queens Square"
-            ],
+            "line": ["Levels 4-6 Barclay House, 37 Queen Square"],
             "city": "London",
             "postalCode": "WC1N 3BH",
             "country": "United Kingdom"
           }
         ]
       },
       "request": {
-        "method": "POST",
-        "url": "Organization"
+        "method": "PUT",
```

Comment:
Is this the correct syntax for a bundle? I thought it was always POST?

Reply:
Yes, it's POST but I was looking at a scenario where we have a repeated report with the same ID. I will resort to the POST.